dask/highlevelgraph.py

```diff
@@ -870,6 +870,49 @@ def merge(cls, *graphs):
         return cls(layers, dependencies)
 
     def visualize(self, filename="dask-hlg.svg", format=None, **kwargs):
```

Comment:
`to_graphviz` in this module could also use a docstring similar to this one; as could `dask.dot.graphviz_to_file`! Maybe not appropriate for this PR.

Reply:
I can add that in a different PR :heavy_check_mark: